mtl.cabal

```diff
@@ -59,24 +52,15 @@ Library
     Control.Monad.Writer.CPS
     Control.Monad.Writer.Lazy
     Control.Monad.Writer.Strict
-  build-depends: base >=4.3 && < 5, transformers >= 0.4 && <0.7
+  
+  build-depends: 
+    , base >=4.12 && < 5
+    , transformers >= 0.5.6 && <0.7
 
-  default-language: Haskell2010
-  other-extensions:
-    CPP
-    MultiParamTypeClasses
-    FunctionalDependencies
-    FlexibleInstances
-    UndecidableInstances
-
-  -- This is a SafeHaskell safeguard (pun intended) to explicitly declare the API contract of `mtl`
-  -- GHC versions before 7.4 were hopelessly broken or incapable of SafeHaskell
-  if impl(ghc >= 7.4)
-    default-extensions: Safe
```

Comment:
Removal of `default-extensions: Safe` may accidentally make some modules unsafe. Please add it back.